libs/fs/src/list_directory.ts

```diff
@@ -52,15 +52,36 @@ export type ListDirectoryError =
   | { type: 'no-entity'; message: string }
   | { type: 'not-directory'; message: string };
 
+/**
+ * Options for listing directories.
+ * - `depth`: How deep to traverse directories. Defaults to 1 (not recursive).
+ * - `includeHidden`: Whether to include hidden files (those starting with a dot or underscore).
```

Comment:
If you put these on the properties themselves then you get the inline docs while writing/hovering over the properties themselves, which I think is more useful than having them here.